src/index.html

```diff
@@ -5,10 +5,59 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Moyo header</title>
+    <title>Flexbox</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+
+      <div class="logo">
+        <a href="#home">
+          <img src="./images/logo.png" alt="logo">
+        </a>
+      </div>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#samsung"class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#smartphones"class="nav__link">Smartphones</a>
+          </li>
```

Comment:
```suggestion
          <li class="nav__item">
            <a href="#apple" class="nav__link is-active">Apple</a>
          </li>

          <li class="nav__item">
            <a href="#samsung" class="nav__link">Samsung</a>
          </li>

          <li class="nav__item">
            <a href="#smartphones" class="nav__link">Smartphones</a>
          </li>
```
Why no spaces between attributes?
Please fix all cases